src/main/java/net/mcreator/ui/MCreator.java

```diff
@@ -407,4 +413,20 @@ public boolean closeThisMCreator(boolean returnToProjectSelector) {
 		return Long.valueOf(windowUID).hashCode();
 	}
 
+	public ActionRegistry getActionRegistry() {

```

Comment:
Now we have a method to access a public-level visibility field, is there any need in this?

Reply:
Read PR description, what is purpose of those comments if you did not check actual PR content?